src/test/java/io/zold/api/WalletTest.java

```diff
@@ -69,9 +69,17 @@ public void mergeIsNotYetImplemented() throws IOException {
         );
     }
 
-    @Test(expected = UnsupportedOperationException.class)
-    public void ledgerIsNotYetImplemented() throws IOException {
-        new Wallet.File(this.folder.newFile().toPath()).ledger();
+    @Test
+    @SuppressWarnings("PMD.AvoidDuplicateLiterals")
+    public void walletShouldBeAbleToReturnLedger() throws Exception {
+        MatcherAssert.assertThat(
+            new Wallet.File(this.wallet(12345L)).ledger(),
+            Matchers.containsInAnyOrder(
+                // @checkstyle LineLengthCheck (2 lines)
+                new RtTransaction("003a;2017-07-19T21:24:51Z;ffffffff9c0ccccd;Ui0wpLu7;98bb82c81735c4ee;For services;SKMPrVj..."),
```

Comment:
@t-izbassar sufficient to just check number of returned transactions here. These are dummy transactions that are invalidly formatted anyhow.